pages/linux/lxc.md

```diff
@@ -0,0 +1,31 @@
+# lxc
+
+> Manage Linux containers (using the lxd REST API).
+
+- List all containers (does not work on public remotes):
+
+`lxc list [{{private-remote}}:][{{filter}}]`
```

Comment:
Is a `private-remote` an IP address? What's a filter? I think that a specific example here might make all the difference.